tests/Moq.Tests/CallbacksFixture.cs

```diff
@@ -490,7 +589,7 @@ public class Derived : Base
 		private void TraceMe(Base b)
 		{
 		}
-		
+
```

Comment:
It's a tiny detail, I know, but could you please revert this whitespace-only change?

Reply:
Automatically done by VS. I reverted it manually using notepad++, hope that now it's OK.